test/unit-tests.c

```diff
@@ -360,28 +360,67 @@ test_dma_controller_remove_region_unmapped(void **state UNUSED)
 static void
 test_dma_map_sg(void **state UNUSED)
 {
-    dma_sg_t sg = { .region = 1 };
-    struct iovec iovec = { 0 };
+    dma_sg_t sg[] = { {.region = 0}, {.region = 1}, {.region = 2} };
+    struct iovec iovec[] = { {0} , {0}, {0} };
```

Comment:
weird extra space here still and below?

Reply:
Oops sorry somehow it got lost in stash clear